addons/delivery/security/ir.model.access.csv

```diff
@@ -11,4 +11,5 @@ access_delivery_carrier_stock_manager,delivery.carrier,model_delivery_carrier,st
 access_delivery_zip_prefix_stock_manager,delivery.zip.prefix,model_delivery_zip_prefix,stock.group_stock_manager,1,1,1,1
 access_delivery_price_rule_stock_manager,delivery.price.rule,model_delivery_price_rule,stock.group_stock_manager,1,1,1,1
 access_choose_delivery_package,access.choose.delivery.package,model_choose_delivery_package,stock.group_stock_user,1,1,1,0
-access_choose_delivery_carrier,access.choose.delivery.carrier,model_choose_delivery_carrier,stock.group_stock_user,1,1,1,0
+access_choose_delivery_carrier,access.choose.delivery.carrier,model_choose_delivery_carrier,sales_team.group_sale_salesman,1,1,1,0
+access_choose_delivery_carrier_stock_user,access.choose.delivery.carrier stock_user,model_choose_delivery_carrier,stock.group_stock_user,1,1,1,0

```

Comment:
Please don't touch the existing rule, as it would break the upgrade. Instead, add the new one (the name doesn't need to be consistent with current versions)
```suggestion
access_choose_delivery_carrier,access.choose.delivery.carrier stock_user,model_choose_delivery_carrier,stock.group_stock_user,1,1,1,0
access_choose_delivery_carrier_sales_team,access.choose.delivery.carrier,model_choose_delivery_carrier,sales_team.group_sale_salesman,1,1,1,0

```

Reply:
Oh, got it 😉  It should be good now